Add students template (#125)
UPB-FILS-MA/upb-fils-ma.github.io: docs/project/2024/your_curs_upb_ro_id.md

# Project Name

:::info 

**Author**: Student Name \
**GitHub Project Link**: link_to_github

:::

## Description

Describe in a few words your project idea.

## Motivation

Why did you choose this project?

## Architecture 

> Add here a schematics with the architecture of you project. Make sure to include:

Reviewer: ```suggestion
Add here the schematics with the architecture of your project. Make sure to include:
```

 - what are the main components (architecture components, not hardware components)
 - how they connect with each other

## Log

<!-- write every week your progress here -->

### Week 6 - 12 May

### Week 7 - 19 May

### Week 20 - 26 May

## Hardware

> Explain in a few words the hardware used.

Reviewer: ```suggestion
Detail in a few words the hardware used.
```

### Schematics

Place your KiCAD schematics here.

### Bill of Materials

<!-- Fill out this table with all the hardware components that you might need.

The format is 
```
| [Device](link://to/device) | This is used ... | [price](link://to/store) |

```

-->

| Device | Usage | Price |
|--------|--------|-------|
| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |

## Software

| Library | Description | Usage |
|---------|-------------|-------|
| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |

## Links

> <!-- Add a few links that got you the idea and that you think you will use for your project -->

Reviewer: ```suggestion
<!-- Add a few links that inspired you and that you think you will use for your project -->
```
